test/JDBC/input/ErrorMapping/TestErrorHelperFunctionsUpgrade-vu-prepare.sql

```diff
@@ -0,0 +1,12 @@
+CREATE VIEW TestErrorHelperFunctionsUpgrade_VU_PREPARE_VIEW AS
+SELECT sql_error_code from sys.fn_mapped_system_error_list();
+GO
+
+create function TestErrorHelperFunctionsUpgrade_VU_PREPARE_FUNC() returns table
+as
+return (select sql_error_code from sys.fn_mapped_system_error_list());
```

Comment:
we should return * instead of sql_error_code, otherwise how will we know if new view is getting called or old one

Reply:
number of rows are also different.. previously, we were sending list of unique tsql error code. And now we are sending whole list which contains same tsql error code twice